dm/dm/worker/source_worker_test.go

```diff
@@ -105,12 +105,19 @@ func (t *testServer) testWorker(c *C) {
 	c.Assert(task, NotNil)
 	c.Assert(task.Result().String(), Matches, ".*worker already closed.*")
 
+	c.Assert(w.StartSubTask(&config.SubTaskConfig{
+		Name: "testStartTask-in-stopped",
+	}, pb.Stage_Stopped, pb.Stage_Stopped, true), IsNil)
+	task = w.subTaskHolder.findSubTask("testStartTask-in-stopped")
+	c.Assert(task, NotNil)
+	c.Assert(task.Result().String(), Matches, ".*worker already closed.*")
```

Comment:
may need discuss with PM, if user creates a stopped task, DM-worker should 
reserve some structure in memory and paused them, or do nothing?

For example, can a stopped incremental task that will read binlog.000001 forbid relay log purging binlog.000001

Reply:
I think the stopped task still have to hold the necessary binlog.

An binlog may relates with multiple tasks. Whatever it's stopped or running , the binlog it needs should not be purged.

Maybe we can offer a feature in future like support query how many tasks relate with this binlog and users can purge it forced.

Reply:
maybe later, current just do nothing